type_traits.hpp

```diff
@@ -0,0 +1,24 @@
+//
+// Created by Lusine Ghazarosyan on 24.04.21.
+//
+
```

Comment:
can't say much on this, trying to learn again (

Reply:
☺️😊